core/testMinimumPhpVersion.php

```diff
@@ -73,8 +73,6 @@
     }
 }
 
-define('PAGE_TITLE_WHEN_ERROR', 'Matomo &rsaquo; Error');
```

Comment:
Any reason for removing that one. If I see that correct, it is still in use...

Reply:
Fat fingers + lack of attention to detail :man_facepalming: 
Thanks for the catch, I've added it back.